### PR TITLE
Create less instances of bn.js

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
Fixed this too.

**Comment by axic** [comment]
@holgerd77 this is ready too

**Comment by holgerd77** [comment]
Rebased this so that this can be merged after review.

File: lib/opFns.js [hunk 1]
```diff
@@ -583,8 +583,8 @@ module.exports = {
     checkOutOfGas(runState, options)
 
     if (!value.isZero()) {
-      runState.gasLeft.iadd(new BN(fees.callStipend.v))
-      options.gasLimit.iadd(new BN(fees.callStipend.v))
+      runState.gasLeft.iaddn(fees.callStipend.v)
+      options.gasLimit.iaddn(fees.callStipend.v)
```
Comment (holgerd77) [inline lib/opFns.js]: Looks good.

File: lib/runBlock.js [hunk 2]
```diff
@@ -203,9 +203,9 @@ module.exports = function (opts, cb) {
   function rewardOmmer (ommer) {
     // calculate reward
     var heightDiff = new BN(block.header.number).sub(new BN(ommer.number))
-    var reward = ((new BN(8)).sub(heightDiff)).mul(minerReward.div(new BN(8)))
+    var reward = ((new BN(8)).sub(heightDiff)).mul(minerReward.divn(8))
 
-    if (reward.lt(new BN(0))) {
+    if (reward.ltn(0)) {
```
Comment (holgerd77) [inline lib/runBlock.js]: Rest looks good as well.

**Pull request merged**